Gopkg.lock

```diff
@@ -3,57 +3,44 @@
 
 [[projects]]
   branch = "dev"
-  digest = "1:8ee7fa4f9c8705684b7a10533435a2b24ceed01ad7d9b238d90d7da29d5c57d7"
   name = "github.com/GianlucaGuarini/go-observable"
   packages = ["."]
-  pruneopts = "NT"
   revision = "d386f0081a66ac3d2c0a79ff4c734dd3165869c0"
 
 [[projects]]
   branch = "master"
-  digest = "1:4cf11742b199ab3aacf26b00187e72f7ff8ba2145f363b74f295d54f098f79e4"
   name = "github.com/agl/ed25519"
   packages = [
     ".",
-    "edwards25519",
+    "edwards25519"
```

Comment:
Please revert your changes to `Gopkg` and upgrade your `dep`.

Reply:
`brew upgrade dep`? @Geod24

Reply:
mine is dep 0.5.0